source/stormwater.rst

```diff
@@ -0,0 +1,329 @@
+
+The model can output potential groundwater recharge volume :math:`VI`, which may also serve as a valuation of retention. However, the model does not currently estimate the pollutant load associated with this recharge volume, as sub-surface transport and transformation of pollutants is not implemented in the model.
+
+Aggregation at the watershed scale (optional)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Users may provide a polygon vector file outlining areas over which to aggregate data (typically watersheds or sewersheds). The model will aggregate the output rasters to compute:
+
+- Average stormwater retention ratio (average of :math:`RE` values)
+- Total retention volume, :math:`m^3` (sum of :math:`VRE` values)
+- Total retained pollutant load for each pollutant, :math:`kg/yr` (sum of :math:`Avoided.load` values)
+- Total pollutant load for each pollutant, :math:`kg/yr` (sum of :math:`Load` values)
+- Total potential recharge volume, :math:`m^3` (sum of :math:`VI`, if infiltration data provided)
+- Total Replacement Cost, currency units (sum of retention costs, if value data provided)
+
+
+Data Needs
+==========
+
+- **Workspace** (required). Folder where model outputs will be written. Make sure that there is ample disk space, and write permissions are correct.
+
+- **Suffix** (optional). Text string that will be appended to the end of output file names, as "_suffix". Use a suffix to differentiate model runs, for example by providing a short name for each scenario. If a suffix is not provided or changed between model runs, the model will overwrite previous results.
+
+- **Land use/land cover (LULC)** (required). Raster map of LULC codes. Each pixel has integer LULC code indicating the landcover type. *All values in this raster MUST have corresponding entries in the biophysical table.*
+
+- **Soil hydrologic group** (required): Raster map of soil hydrologic groups. Each pixel's value is 1, 2, 3, or 4, corresponding to hydrologic soil groups A, B, C, and D respectively.
+
+- **Precipitation** (required). Raster map of annual precipitation in :math:`mm/yr`
+
+- **Biophysical table** (required). A .csv (Comma Separated Value) table containing model information corresponding to each of the land use classes in the LULC raster. *All LULC classes in the LULC raster MUST have corresponding values in this table.* Each row is a land use/land cover class and columns must be named and defined as follows:
+
+    - **lucode**: Unique integer for each LULC class (e.g., 1 for forest, 3 for grassland, etc.) *Every value in the LULC map MUST have a corresponding lucode value in this column.*
+
+    - **is_impervious** (only required if **Adjust retention ratios** is checked: Binary value indicating whether each LULC class is directly-connected impervious (enter 1 for yes, 0 for no). This is used to adjust the retention coefficients for nearby pixels.
```

Comment:
Feedback from Milan:
> For the ‘is impervious’ column, it might need to be a bit more clear when a LULC is considered to be impervious enough to be labelled as 1 for the model. Is every class that is not considered 100% pervious (e.g. grassland) considered pervious? (e.g. very low density urban fabric with 20% perviousness?) Or is it meant to ‘assume’ every urbanized area as ‘impervious’ due to their likely connection to the sewage system? (I already adjusted the RC and IR values for each LULC depending on their avg. imperviousness so that is not an issue)

Ben said via email:
> This is where the documentation could use some more detail, as the consideration of connectedness is somewhat subjective, but also tied to the land cover layer being used. We developed it with the US National Land Cover Database (NLCD) in mind, which has some broad "developed" categories (Open, Low, Medium, and High) that are essentially bins of imperviousness. We further made the assumption that the "High" and "Medium" categories would likely be fully connected to storm sewer systems (due to having > 50% imperviousness per the NLCD specification), while the Low and Open categories were lower imperviousness (< 50%) and might contain some cover types like parks that are not connected and/or provide more retention. We left the determination of "connectedness" up to the user, since land cover layers might have more (or less) detail, depending on where the model is being used.

This is great info to include.